maths/perfect_cube.py

```diff
@@ -1,11 +1,27 @@
 def perfect_cube(n: int) -> bool:
     """
-    Check if a number is a perfect cube or not.
+      Check if a number is a perfect cube or not.
 
     >>> perfect_cube(27)
-    True
-    >>> perfect_cube(4)
-    False
+      True
+      >>> perfect_cube(4)
+      False
+      >>> perfect_cube(8)
+      False
+      >>> perfect_cube(125)
+      True
+      >>> perfect_cube(64)
+      True
+      >>> perfect_cube(0)
+      True
+      >>> perfect_cube(1)
+      True
+      >>> perfect_cube(3.375)
```

Comment:
These three tests should raise ValueError

Reply:
can you have a look now?

Reply:
Below there is a ❌ next `build` test to say that it failed.  Click on the `Details` link to the right of that line.  The test failure is at about line 807 that outputs.

Reply:
Fixed @cclauss